docs/userguide/canvas.rst

```diff
@@ -941,7 +941,12 @@ Chords may have callback and errback signatures linked to them, which addresses
 some of the issues with linking signatures to groups.
 Doing so will link the provided signature to the chord's body which can be
 expected to gracefully invoke callbacks just once upon completion of the body,
-or errbacks just once if any task in the chord header or body fails.
```

Comment:
I believe this was a mistake in the documentation, as the implementation only links the body to the error before this patch

Reply:
Are we sure about this? If we run the same integration test without our feature flag enabled does no errback get called?

Reply:
You were right, I was wrong. Change in doc will be reverted, original doc validates as correct (new test added to confirm).

Reply:
Fix pushed